src/lib/broadcast_node/outgoing_message.ts

```diff
@@ -1,8 +1,8 @@
-import { VirtualValueID } from "zwave-js";
+import { VirtualValueID, SetValueResult } from "zwave-js";
 import { BroadcastNodeCommand } from "./command";
 
 export interface BroadcastNodeResultTypes {
-  [BroadcastNodeCommand.setValue]: { success: boolean };
+  [BroadcastNodeCommand.setValue]: { result: SetValueResult };
```

Comment:
@raman325  how do you usually handle return type changes? Just like this or do we need to decide on the schema version?